src/Blocks/BlockEnchantmentTable.h

```diff
@@ -34,6 +34,11 @@ class cBlockEnchantmentTableHandler :
 		UNUSED(a_Meta);
 		return 29;
 	}
+
+	virtual double GetBlockHeight(void) override
+	{
+		return 12.0 / 16.0;
```

Comment:
So every time, you use this method, this is being calculated instead of putting this into a constant. Generally what about making a constant list of all the block heights?